hyperspy/_components/eels_cl_edge.py

```diff
@@ -69,27 +69,45 @@ class EELSCLEdge(Component):
         favourable cases is proportional to the number of atoms of
         the element. It is a component.Parameter instance.
         It is fixed by default.
-    fine_structure_coeff : Parameter
-        The coefficients of the spline that fits the fine structure.
-        Fix this parameter to fix the fine structure. It is a
-        component.Parameter instance.
     effective_angle : Parameter
         The effective collection semi-angle. It is automatically
         calculated by set_microscope_parameters. It is a
         component.Parameter instance. It is fixed by default.
-    fine_structure_smoothing : float between 0 and 1
+    fine_structure_active : bool, default False
+        Activates/deactivates the fine structure features. When active,
+        the fine structure region model is not defined by the simulated
+        EELS core-loss edge, but by a spline (if `fine_structure_spline`
+        is `True`) and/or any component in `fine_structure_components`.
+    fine_structure_spline : bool, default True
+        If True and `fine_structure_active` is True, the region from
+        `fine_structure_spline_onset` until `fine_structure_width`
+        are modeled with a spline.
+    fine_structure_coeff : Parameter
+        The coefficients of the spline that fits the fine structure.
+        Fix this parameter to fix the fine structure. It is a
+        component.Parameter instance.
+    fine_structure_smoothing : float between 0 and 1, default 0.3
         Controls the level of smoothing of the fine structure model.
         Decreasing the value increases the level of smoothing.
-    fine_structure_active : bool
-        Activates/deactivates the fine structure feature.
+    fine_structure_spline_onset : float, default 0.
+        The position, from the ionization edge onset, at which the region
+        modeled by the spline function starts.
+    fine_structure_width : float, default 30.
+        The width of the region, from the ionization edge onset, where
```

Comment:
probably 'from the fine structure spline onset'? Also, the model is given the function/component or the data is modelled in that region?

Reply:
No, it is correct as is: it defines the fine structure width, regardless of the fine structure kind (spline or components).

Second question: in that region we do not use the EELS cross-section to fit the data. Instead, we fit it with a spline and/or the components in `fine_structure_components`.

The sentence wasn't clear, I've tried to improve it.